### PR TITLE
zh-trans: modify /docs/reference/glossary/containerd.md

### PR DESCRIPTION
**Pull request by lpf7551321**
zh-trans: modify /docs/reference/glossary/containerd.md
Update: /docs/reference/glossary/containerd.md
Co-Authored-By: lpf7551321

### CODE REVIEW

File: content/zh/docs/reference/glossary/containerd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+aka:
+tags:
+- tool
+---
+ A container runtime with an emphasis on simplicity, robustness and portability
+
+<!--more-->
+
+containerd is a {{< glossary_tooltip text="container" term_id="container" >}} runtime
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/containerd.md]: 为什么没有翻译呢？

Comment (lpf7551321) [reply content/zh/docs/reference/glossary/containerd.md]: pr提错了，已修改

File: content/zh/docs/reference/glossary/containerd.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: containerd
+id: containerd
+date: 2019-05-14
+full_link: https://containerd.io/docs/
+short_description: >
+ 强调简单性，健壮性和可移植性的容器运行时
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/containerd.md]:
```suggestion
 强调简单性、健壮性和可移植性的一种容器运行时
```

File: content/zh/docs/reference/glossary/containerd.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+<!--more-->
+
+<!--
+containerd is a {{< glossary_tooltip text="container" term_id="container" >}} runtime
+that runs as a daemon on Linux or Windows. containerd takes care of fetching and
+storing container images, executing containers, providing network access, and more.
+-->
+
+containerd 是一种 {{< glossary_tooltip text="container" term_id="container" >}} 能在 Linux 或者 Windows 后台运行的运行时。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/containerd.md]:
```suggestion
containerd 是一种 {{< glossary_tooltip text="容器" term_id="container" >}} 运行时，能在 Linux 或者 Windows 后台运行。
```

File: content/zh/docs/reference/glossary/containerd.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+containerd 能运输、存储容器镜像，执行容器实例，提供网络访问等。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/containerd.md]:
```suggestion
containerd 能取回、存储容器镜像，执行容器实例，提供网络访问等。
```

**Pull request merged**